tectonic/dpx-vf.c

```diff
@@ -75,6 +75,14 @@ static struct vf *vf_fonts = NULL;
 static unsigned int num_vf_fonts = 0;
 static unsigned int max_vf_fonts = 0;
 
+
+void vf_reset_global_state(void)
```

Comment:
Very minor point, I like to a newline between the function name and the return type. That way you can pull up the function definition with `git grep ^vf_reset_global_state`.